pkgs/desktops/gnome/core/nautilus/default.nix

```diff
@@ -52,6 +53,17 @@ stdenv.mkDerivation rec {
       src = ./fix-paths.patch;
       inherit tracker;
     })
+
+    # Fix build with latest libportal
+    # https://gitlab.gnome.org/GNOME/nautilus/-/merge_requests/749
+    (fetchpatch {
+      url = "https://gitlab.gnome.org/GNOME/nautilus/-/commit/55cfd66ccca391fc144f5863ff6bfc1f3b137e2d.patch";
+      sha256 = "xSb9l7xxEYpAwmdmeWT/t7Z9Ck3DPtsODzbReQW/Q70=";
+      excludes = [
+        "build-aux/flatpak/org.gnome.Nautilus.json"
+        "build-aux/flatpak/org.gnome.Nautilus.yml"
```

Comment:
why exculding these two files?

Reply:
Due to merge conflicts and irrelevancy.

Reply:
Ok I can see now that building flatpaks is irrelevant for us. Thanks for explaining.